### DISCUSSION TITLE
Error on OverlayPanel

### DISCUSSION THREAD

**Discussion by Charl20cas** [primefaces/community#312] [PrimeNG] [answered]
Hi everyone!
I have a problem with this component in Angular15+PrimeNG15. When there are several elements that open the same component, the overlay toogle function gives a javascript error

ERROR TypeError: Cannot read properties of null (reading 'stopPropagation')
    at OverlayPanel.show (primeng-overlaypanel.mjs:97:25)
    at OverlayPanel.destroyCallback (primeng-overlaypanel.mjs:87:26)
    at OverlayPanel.onAnimationEnd (primeng-overlaypanel.mjs:177:26)
    at OverlayPanel_div_0_Template_div_animation_animation_done_0_listener (primeng-overlaypanel.mjs:289:32)
    at executeListenerWithErrorHandling (core.mjs:13975:16)
    at wrapListenerIn_markDirtyAndPreventDefault (core.mjs:14008:22)
    at animations.mjs:192:25
    at Array.forEach (<anonymous>)
    at animations.mjs:190:52
    at _ZoneDelegate.invoke (zone.js:412:30)
    
    
 An example:
 
HTML
<button label="Option1" (click)="moreMenu($event, panel)" >
<button label="Option2" (click)="moreMenu($event, panel)" >

<p-overlayPanel #panel> HOLA </p-overlayPanel>
 
 TYPESCRIPT
public moreMenu(event: any, overlay: OverlayPanel) {
    if (event) {
      event.stopPropagation();
    }
    overlaypanel.toggle(event);
}

 This same example in angular 10 + PrimeNG10 works correctly.

Any idea?
Thanks!!

**Comment by cetincakiroglu** [comment]
Could you please create a stackblitz demo so we can identify the issue clearly?

**Reply by Charl20cas** [reply]
https://stackblitz.com/edit/e3fkqm?file=src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.ts

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

Cannot reproduce the issue with the latest PrimeNG(16.6.0), it's probably fixed between v15-v16.6.0

Here is an example with the latest version https://stackblitz.com/edit/ujogis?file=src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.html,src%2Fapp%2Fdemo%2Foverlay-panel-basic-demo.ts

**Reply by Charl20cas** [reply]
It's a shame, the error may not be reproduced with primeng16, but the functionality is not the same as in primeng9. In primeng9, clicking on button 1 showed the ovelaypanel, then, pressing button 2, the ovelaypanel of button 1 is hidden and the ovelaypanel of button 2 is automatically shown. Thank you anyway.

Example: https://www.primefaces.org/primeng-v9-lts/#/overlaypanel (Table Integration)

**Comment by dextreti** [comment]
![error-prime-ng](https://github.com/primefaces/community/assets/147953503/a9f9d1ff-22ec-409d-b07d-70250f986992)
![error-prime-ng-222](https://github.com/primefaces/community/assets/147953503/368ed8b3-3307-4e95-9403-75154ff2578d)
